api/aperture/common/peers/v1/peers.proto

```diff
@@ -3,14 +3,21 @@ syntax = "proto3";
 package aperture.common.peers.v1;
 
 import "google/api/annotations.proto";
+import "google/protobuf/empty.proto";
 
 //
 // gRPC service
 //
 
 // PeerDiscoveryService is used to query Peers.
 service PeerDiscoveryService {
-  rpc GetPeers(PeersRequest) returns (Peers) {
+  rpc GetPeers(google.protobuf.Empty) returns (Peers) {
+    option (google.api.http) = {
+      get: "/v1/peers"
+    };
+  }
+
+  rpc GetPeer(PeerRequest) returns (Peer) {
```

Comment:
Can the same API use to get both all the peers and specific peer was the question we had here - I ended up having repeated address field for the PeerRequest, is there different approach we can try?

Reply:
repeated address field is not a good fit here. We need to get a single peer at a time and the peer database is not a nested map unlike status registry. e.g. for status we have nested keys (hence the repeated field) -- /v1/status/liveness/watchdog/heap and so on.